src/rtcModule/webrtcAdapter.cpp

```diff
@@ -51,7 +54,18 @@ bool init(void *appCtx)
     threadMgr->SetCurrentThread(thread);
 
     rtc::InitializeSSL();
-    gWebrtcContext = webrtc::CreatePeerConnectionFactory();
+    if (gWebrtcContext == nullptr)
+    {
+        gWebrtcContext = webrtc::CreatePeerConnectionFactory(
```

Comment:
We should review the thread's management here, to ensure that it will not break the existing synchronization between MEGAchat's thread and WebRTC's thread/s.